lua/mason-registry/gospel/init.lua

```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local go = require "mason-core.managers.go"
+
+return Pkg.new {
+    name = "gospel",
+    desc = [[misspelled word linter for Go comments, string literals and embedded files]],
+    homepage = "https://github.com/kortschak/gospel",
+    categories = {},
```

Comment:
This seems to be a linter, no?

```suggestion
    categories = { Pkg.Cat.Linter },
```

Reply:
yes